src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -188,17 +192,10 @@ public ScanDetails executeCxScan(ScanRequest request, File cxFile) throws Machin
 
             scanId = cxService.createScan(params, getComment(request));
 
-            BugTracker.Type bugTrackerType = bugTrackerEventTrigger.triggerBugTrackerEvent(request);
-            if (bugTrackerType.equals(BugTracker.Type.NONE)) {
-                return handleNoneBugTrackerCase(request, cxFile, scanId, projectId);
-            } else {
-                cxService.waitForScanCompletion(scanId);
-                projectId = handleUnKnownProjectId(projectId, ownerId, request.getProject());
-                osaScanId = createOsaScan(request, projectId);
+            osaScanId = createOsaScan(request, projectId);
```

Comment:
the scope of osaScanId can be limited to this else statement

Reply:
it can be written as:
Optional.ofNullable(createOsaScan(request, projectId))
.ifPresent( osaScanId -> logRequest(request, osaScanId, cxFile, OperationResult.successful()) )